spec/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative "../lib/User"
+
+describe User do
+  let(:menu) {double(:menu,{starters: {"Soup" => ["Soup",3.99], "Salad" => ["Salad",4.99]}})}
```

Comment:
Metrics/LineLength: Line is too long. [93/80]<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.